docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -195,11 +235,18 @@ configured during the initial integration enrollment flow and can be adjusted
 as necessary after the initial sync completes.
 
 Each imported Access List is automatically assigned a role (or a set of roles)
-that grant all members of that list access to a particular permission set on a
-particular AWS account based on the permissions the corresponding Identity Center
-group was assigned during the integration setup. Those roles are considered
-system roles generated by Teleport and are named using `<permission-set-name>-on-<account-name>`
-convention (e.g. `AdministratorAccess-on-my-account`).
+that grant all members of that list access to all of the Account Assignments
+assigned to the corresponding AWS Identity Center group during the integration
+setup. 
+
+These Teleport-generated roles each represent a single Account Assignment, and
```

Comment:
I think that we have replaced permission-set-name>-on-<account-name to permission-set-name>-on-<account-name>-aws-coount-id to prevent collisions between account names